lib/action_cable/subscription_adapter/solid_cable.rb

```diff
@@ -104,8 +104,18 @@ def broadcast_messages
             end
 
             def with_polling_volume
```

Comment:
Lets have this more mirror solid queues implementation:
```ruby
          if ::SolidCable.silence_polling? && ActiveRecord::Base.logger
            ActiveRecord::Base.logger.silence { yield }
          else
            yield
          end
```

Reply:
It was at the start, but when i was running it on a Rails 7.2.1 application silence was not defined inside ActiveRecord::Base

Reply:
If you prefer i can do a :
`if Rails.version.to_f < 8.0`

Reply:
Hmm on 7.2 this is working for me:
<img width="773" alt="image" src="https://github.com/user-attachments/assets/8f0c3523-7cfe-466d-8a24-e7f4676c65c6">

Whats your app setup like?

Reply:
Yeah my mistake i was setting up in my config/environments/development.rb
`config.active_record.logger = Logger.new($stdout)`
instead of 
```
config.logger = ActiveSupport::Logger.new($stdout)
    .tap  { |logger| logger.formatter = Logger::Formatter.new }
    .then { |logger| ActiveSupport::TaggedLogging.new(logger) }
```

All good you are right i will fix the code in the PR